File: Programmer_test/PT_Python/P12930.py
def solution(s):
    answer = ''
    temp = 0
    for i in s:
        if (i == ' '):
            answer += ' '
            temp = 0
        else:
            if temp % 2 == 0:
                answer += i.upper()
            else:
                answer += i.lower()
            temp += 1
    return answer

print(solution("try hello world"))

File: Programmer_test/PT_Java/P12930.java
public class P12930 {
    public static void main(String[] args) {
        P12930 T = new P12930();
        System.out.println(T.solution("try hello world"));
    }

    public String solution(String s) {
        StringBuilder sb1 = new StringBuilder();
        int temp = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == ' ') {
                temp = 0;
                sb1.append(" ");
            }
            else {
                if ((temp % 2) == 0) {
                    sb1.append(Character.toUpperCase(s.charAt(i)));
                }
                else {
                    sb1.append(Character.toLowerCase(s.charAt(i)));
                }
                temp += 1;
            }
        }
        return sb1.toString();
    }
}